### PR TITLE
Update of fedmsgs which are more accurate to what we publish

### PR DESCRIPTION
**Pull request by arilivigni**
- Updated fields
- Everything in message properties
- message context is blank

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,422 +203,566 @@ email=jchaloup@redhat.com
...
+  "i": 1,
+  "timestamp": 1496382015, 
+  "msg_id": "2017-f2b46ca1-1b07-4539-8d86-f7d3cb1b5934", 
+  "crypto": "x509", 
+  "topic": "org.centos.prod.ci.pipeline.test.functional.queued",   
+  'msg': { 
+        "CI_TYPE": "custom", 
+        "CI_NAME": "ci-pipeline-atomic-host-tests",
+        "topic": "org.centos.prod.ci.pipeline.test.functional.queued",
+        "build_id": "79",
```
Comment (johnbieren) [inline README.md]: do we want package_url in here?

File: README.md [hunk 2]
```diff
@@ -203,422 +203,566 @@ email=jchaloup@redhat.com
...
+  "i": 1,
+  "timestamp": 1496382015, 
+  "msg_id": "2017-f2b46ca1-1b07-4539-8d86-f7d3cb1b5934", 
+  "crypto": "x509", 
+  "topic": "org.centos.prod.ci.pipeline.test.functional.running",   
+  'msg': { 
+        "CI_TYPE": "custom", 
+        "CI_NAME": "ci-pipeline-atomic-host-tests",
+        "topic": "org.centos.prod.ci.pipeline.test.functional.running",
+        "build_id": "79",
```
Comment (johnbieren) [inline README.md]: do we want package_url in here?

File: README.md [hunk 3]
```diff
@@ -203,422 +203,566 @@ email=jchaloup@redhat.com
...
+  "msg_id": "2017-f2b46ca1-1b07-4539-8d86-f7d3cb1b5934", 
+  "crypto": "x509", 
+  "topic": "org.centos.prod.ci.pipeline.package.running",   
+  'msg': { 
+        "CI_TYPE": "custom", 
+        "CI_NAME": "ci-pipeline-rpmbuild",
+        "topic": "org.centos.prod.ci.pipeline.package.running",
+        "build_id": "79", 
+        "build_url": "https://ci.centos.org/job/ci-pipeline-rpmbuild/79/",
+        "package_url": "<full-url-to-package>",
```
Comment (johnbieren) [inline README.md]: These messages kick off when the rpmbuild job starts, I don't think we have package_url until the .complete message

Comment (arilivigni) [inline README.md]: Yes just a copy and paste issue I will adjust it

File: README.md (hunk 1 above)
Comment (arilivigni) [inline README.md]: nope

File: README.md (hunk 2 above)
Comment (arilivigni) [inline README.md]: no I will remove it

**Pull request merged**